content/source/docs/enterprise/admin/update-tfe-license.html.md

```diff
@@ -0,0 +1,161 @@
+
+1.  Connect to the Terraform Enterprise host machine using SSH.
+
+2.  Examine the output of the command `docker logs replicated` and look for the warning message below.
+
+         WARN 2021-02-22T01:40:00+00:00 tasks/app_tasksteps.go:113 Airgap license on disk does not match installed license
+
+3.  If this warning message occurs, it indicates that the incorrect license file is being used. Please contact your Customer Success Manager if you have one assigned to your account, or contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) for further assistance. When contacting support, please include the output from the `replicatedctl license inspect` command after running it on the existing Terraform Enterprise host machine.
+
+### Unable to sync license: Error: Unsuccessful HTTP response
+
+The error message below may occur due to several possible issues.
+
+    Unable to sync license: Error: Unsuccessful HTTP response
+
+*   Network communication to the Replicated servers
+
+    This symptom can happen with the online installation type when the current network infrastructure has changed since the installation of Terraform Enterprise. Terraform Enterprise requires network communication to the Replicated endpoints, as noted in the [network requirements](https://www.terraform.io/docs/enterprise/before-installing/network-requirements.html) guideline. With the license sync operation, the Replicated installer will attempt to contact `api.replicated.com` in order to retrieve the license information. The specific IP addresses of Replicated services for the Terraform Enterprise can be found in [Replicated’s GitHub repository](https://github.com/replicatedhq/ips/blob/master/ip_addresses.json).
+
+*   Mismatched customer name between the installed license and the new license.
+
+    This symptom can happen with the Airgap installation type in some conditions, for example, when a trial license was issued with a customer name and the new paid license was issued on a different customer name. If you are on the Airgap installation and encountered this error, please contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) together with the output from the command `replicatedctl license inspect` from the existing Terraform Enterprise host machine to request for further assistance.
+
+### Incorrect version of airgap file
+
+The error message below indicates the incorrect version of airgap file being used against the installed version of Terraform Enterprise.
+
+    installed app release (325b33bf0ad539c994644423128cad5e:502) does not match the airgap package
```

Comment:
Wrapped in code fence.

````suggestion
    ```plaintext
    installed app release (325b33bf0ad539c994644423128cad5e:502) does not match the airgap package
    ```
````